### DISCUSSION TITLE
gtag event

### DISCUSSION THREAD

**Discussion by Visualapproch** [AmauriC/tarteaucitron.js#821] [Q&A]
Bonjour

J'ai un évènement gtag event sur un bouton d'envoi de formulaire Contact Form 7 codé de la façon suivante  :
`<script>document.addEventListener( 'wpcf7mailsent', function( event ) {
    	gtag('event', 'envoi_formulaire_contact', {'event_category': 'prospect', 'event_label': ''});
    	}, false );</script>`
Il me semble avoir compris que je dois appeler cet événement avec gtaMore?
Mais j'utilise déjà gtaMore :
`tarteaucitron.user.gtagMore = function () { gtag('config', 'UA-XXXXXX-1', {'decorate_forms': true}) };`
Comment je peux rajouter la détection de mon évènement ?

Merci

**Comment by AmauriC** [comment]
Il n'y a rien à faire pour cet event.

Si GA est activé sur tarteaucitron, la fonction `gtag` sera appelé et l'event remontera normalement.

**Reply by Visualapproch** [reply]
Je n'ai pas l'impression que cela fonctionne ...

https://www.espaceblueocean.com/contact-camping-blue-ocean-ondres-capbreton/

**Reply by AmauriC** [reply]
Après avoir activé GA sur tarteaucitron et executé via la console:
`gtag('event', 'envoi_formulaire_contact', {'event_category': 'prospect', 'event_label': ''});`
J'ai bien un event GA qui est envoyé à google.

**Comment by Visualapproch** [comment]
Merci
Il est possible que le problème vienne de la position du script qui était positionné avant le script d'activation de GA sur tarteaucitron.
Nous allons vérifier.
